noetic/distribution.yaml

```diff
@@ -2754,6 +2754,12 @@ repositories:
       url: https://github.com/ros-drivers/four_wheel_steering_msgs.git
       version: master
     status: maintained
+  foxglove_bridge:
+    release:
```

Comment:
I would recommend adding source and doc entries as wells

Reply:
https://github.com/ros/rosdistro/pull/35384/commits/025d1ea27f33a4dee5338add6fa8e28671b67ef8